OpenRA.Mods.Common/Traits/BlocksProjectiles.cs

```diff
@@ -13,10 +13,11 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
-	// TODO: Add functionality like a customizable Height that is compared to projectile altitude
 	[Desc("This actor blocks bullets and missiles with 'Blockable' property.")]
 	public class BlocksProjectilesInfo : UpgradableTraitInfo
 	{
+		public readonly WDist Height = WDist.FromCells(6);
```

Comment:
> Pretty sure a height of 1 WDist should be enough for the first 3 games.

Unless you meant 1 cell (1024 WDist), no, definitely not. For example, the cannon armament's LocalOffset Z value for mammoth tanks is 340 WDist above ground, so even at half the distance towards target the altitude is still 170 WDist above ground. The weapon's launch offset is factored in, so even straight-flying projectiles usually won't fly that low until right in front of the target.

That being said, in hindsight 6 cells is certainly over the top, I guess 1 cell/1024 WDist should be enough.